src/test/ui/issues/issue-82156.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    super(); //~ ERROR
```

Comment:
```suggestion
    super(); //~ ERROR failed to resolve: there are too many leading `super` keywords
```
Could you also move this test from `ui/issues` to `ui/resolve`?

Reply:
I tried adding this message but for some reason the test runner wouldn't accept it. I'll try again.

Reply:
@petrochenkov what is the difference between 'resolve' and 'issues'?

Reply:
@osa1 
Grouping tests into the `issues` subdirectory is meaningless.
The general idea is for test subdirectories to be related to specific compiler or language areas, so they could have "owners" from the corresponding maintainers and working groups.
Also see https://github.com/rust-lang/rust/issues/73494 and related issues.